packages/metadb/metadb.1.0.1/opam

```diff
@@ -34,9 +34,9 @@ build: [
 ]
 dev-repo: "git+https://github.com/nguermond/doculib.git"
 url {
-  src: "https://github.com/nguermond/doculib/archive/v1.3.1.tar.gz"
+  src: "https://github.com/nguermond/doculib/archive/v1.3.2.tar.gz"
```

Comment:
mmmh, could you release `metadb.1.0.2` instead?

Reply:
I split the packages to different repositories, sorry about the force push